src/mesh/mesh_base.C

```diff
@@ -96,6 +97,7 @@ MeshBase::MeshBase (const MeshBase & other_mesh) :
   _skip_noncritical_partitioning(false),
   _skip_all_partitioning(libMesh::on_command_line("--skip-partitioning")),
   _skip_renumber_nodes_and_elements(false),
+  _skip_find_neighbors(false),
```

Comment:
Looks like you followed what was done for the `_skip_renumber_nodes_and_elements` parameter, but I wonder if both of these shouldn't get their values from `other_mesh`?

Reply:
Thanks for fixing both of these.

Reply:
:+1: